src/main/webapp/script.js

```diff
@@ -736,6 +736,3 @@ function transformAgregatedDataforGeoChart(aggregatedData) {
     google.charts.setOnLoadCallback(drawRegionsMap(array, aggregatedData)); 
   });
 }
-
-module.exports = {getTotalCosts, getAverageCosts, getFailedJobs, getFailedJobsCost,
```

Comment:
As I explained earlier, perhaps not very well, this needs to be in the code for testing. Whilst it doesn't do anything for the the web application itself (hence the error message that module doesn't exist), it allows the tests in the script.test.js file to run (they won't run without this line of code) - it's part of the testing framework.

Reply:
Sorry for removing this line. I was trying to solve the error and forgot to add it back :D

Reply:
No worries! I've added a comment above with the code back in so you should just be able to commit that (not entirely convinced the indentation is correct though)